extras/configure_gluster_collectd.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Install and configure gluster collectd package
+  hosts: all
+  vars:
+    copr_url: "https://copr-be.cloud.fedoraproject.org/results/redara"
+    repo: "gluster-collectd/fedora-27-x86_64/00774074-gluster-collectd"
+    package: "gluster-collectd-1.0.0-1.fc27.noarch.rpm"
+    gluster_collectd_rpm: "{{ copr_url }}/{{ repo }}/{{ package }}"
+    collectd_conf_dir: "/etc/collectd.d"
+    hostname: "{{ brick_ip | default(ansible_hostname) }}"
+  tasks:
+    - name: checking dependency packages are present
+      yum:
+        name: "{{ item }}"
```

Comment:
You can give a list to the name. In this case the module will be run three times.

For eg:

pkgs:
  - collectd
  - collectd-python
  - python-psutil

yum:
    name: "{{ pkgs }}"